db/migrate/20170222214902_add_initiator_to_service.rb

```diff
@@ -0,0 +1,14 @@
+class AddInitiatorToService < ActiveRecord::Migration[5.0]
+  class Service < ActiveRecord::Base; end
+  def up
+    add_column :services, :initiator, :string
+    change_column_comment :services, :initiator, "Record who initiated the service creation user|control|alert. It helps the UI display the services based on categories"
```

Comment:
I would remove the UI reference entirely.  Additionally, I don't think we should encode the enum in the comment, because like most code comments, it is highly likely to get stale if there become other types of initiators  Perhaps `"Entity that initiated the service creation"` ?